### PR TITLE
RSS-K8: Adding environment variable to get the RSS server in K8 environment

### PR DESCRIPTION
**Pull request by mabansal**
We need to discover the RSS server, in this diff we are adding an environment variable to discover that.

### CODE REVIEW

**Review by mayurdb — APPROVED** [review]

**Comment by mayurdb** [comment]
@mabansal  Can we add a small section in the README of the base branch around the working of RSS servers in K8s -> server ids convention, around plans for the service registry, how would clients discover the RSS servers, etc

File: src/test/scala/org/apache/spark/shuffle/TestUtil.scala [hunk 1]
```diff
@@ -47,4 +47,15 @@ object TestUtil {
     conf.set("spark.shuffle.rss.serviceRegistry.zookeeper.servers", zooKeeperServers)
     conf
   }
+
+  def newSparkConfWithSequenceServerRegistryServer(appId: String): SparkConf ={
+    val conf = TestUtil.newSparkConfWithStandAloneRegistryServer(appId, "")
+    var testStreamServer: TestStreamServer =
```
Comment (vectorijk) [inline src/test/scala/org/apache/spark/shuffle/TestUtil.scala:53]: it shall be immutable. maybe change `var` to `val`
```suggestion
    val testStreamServer: TestStreamServer =
```

**Review by vectorijk — APPROVED** [review]
lgtm

File: src/test/scala/org/apache/spark/shuffle/TestUtil.scala (hunk 1 above)
Comment (mabansal) [reply src/test/scala/org/apache/spark/shuffle/TestUtil.scala:53]: > it shall be immutable. maybe change `var` to `val`

Done

**Comment by mabansal** [comment]
> @mabansal Can we add a small section in the README of the base branch around the working of RSS servers in K8s -> server ids convention, around plans for the service registry, how would clients discover the RSS servers, etc

I will add that part of README in K8 branch to avoid confusion

**Comment by mabansal** [comment]
> > @mabansal Can we add a small section in the README of the base branch around the working of RSS servers in K8s -> server ids convention, around plans for the service registry, how would clients discover the RSS servers, etc
> 
> I will add that part of README in K8 branch to avoid confusion

Done

**Comment by mabansal** [comment]
Mergin in

**Pull request merged**